how-to-guide/facilitating.md

```diff
@@ -0,0 +1,70 @@
+# How to run a Data Ethics Club meeting
+
+Typically we have a brief introduction to the material, split everyone into breakout rooms of 5-6 people for 25-30 minutes to talk about the discussion questions, and then come back together for the final 15-20 minutes to share thoughts before voting on the material for the next meeting. 
+
+We use the collaborative markdown site [HackMD](https://hackmd.io/) to collect everyone's views on the content whilst we're in breakout rooms. Our templates for this are linked below.  
+
+## Roles and getting involved
+We tend to have separate responsibilities split into facilitating and organising, and we swap these roles around each meeting.  
+The **facilitator** is responsible for leading on the content,  managing the discussion in the meeting and writing up the summary of the meeting.  
+The **organiser** is responsible for sending out emails, setting up the templates and managing participants and breakout rooms in the meeting.  
```

Comment:
I think the term organiser might be a bit confusing here, because we also use the word "organisers" for the kind of general group decision-making, i.e. what to vote on. No better word immediately comes to mind, but let's think about this!